porting/debug-build/dmesg.rst

```diff
@@ -0,0 +1,13 @@
+
+Reading kernel logs
+===================
+
+To find out what happened during an unsuccessful boot you can check the kernel log files. These can usually be retrieved from ``/proc/last_kmsg`` after rebooting the device into another, working system. A precondition for this is that you have a system that you can successfully boot, such as TWRP.
```

Comment:
I would also mention that this file is not available for all kernel versions. I was really confused myself when I couldn't find it on my device